pkg/storage/postgresql/targets.go

```diff
@@ -202,24 +239,37 @@ func (db DB) CreateTarget(t model.Target) (model.Target, error) {
 		INSERT INTO targets (identifier)
 		SELECT $1
 		WHERE NOT EXISTS (SELECT 1 FROM target_exists)
-		RETURNING *, ARRAY[]::text[] AS tags
+		RETURNING *, ARRAY[]::text[] AS teams, ARRAY[]::text[] AS tags
 	)
 	SELECT * FROM target_exists
     UNION ALL
 	SELECT * FROM creation_query`
 	err = tx.QueryRow(query, t.Identifier).Scan(
-		&created.ID, &created.Identifier, &tags,
+		&created.ID, &created.Identifier, &teams, &tags,
 	)
 	if err != nil {
 		tx.Rollback()
 		return model.Target{}, err
 	}
 
+	// Create associated teams if any
```

Comment:
Just nitpicking, but should we finish this comment with a period?

Reply:
Aggreed, fixed in [dc196d5](https://github.com/adevinta/vulnerability-db-api/pull/42/commits/dc196d54a6b48e6ab982f3177718833eec1b09fb).